staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1689,6 +1709,14 @@ const (
 	MountPropagationBidirectional MountPropagationMode = "Bidirectional"
 )
 
+// volumeDevice describes a mapping of a raw block device within a container.
+type VolumeDevice struct {
+	// Required: This must match the Name of a Volume in the pod.
```

Comment:
// name must match the name of a persistentVolumeClaim volume in the pod

Reply:
ack...fixed